src/NoteMouseHandler/NoteMouseHandler.js

```diff
@@ -0,0 +1,97 @@
+function findMap(arr, func) {
+  for (const item of arr) {
+    const result = func(item)
+    if (result) return result
+  }
+  return null
+}
+
+export default class MouseHandler {
```

Comment:
__[jshint]__
*  'export' is only available in ES6 (use 'esversion
*  'class' is available in ES6 (use 'esversion

<!-- Comment by [SideCI](https://sideci.com) -->